easybuild/easyblocks/generic/intelbase.py

```diff
@@ -434,12 +434,6 @@ def make_module_extra(self, *args, **kwargs):
             txt += self.module_generator.prepend_paths(self.license_env_var, [self.license_file],
                                                        allow_abs=True, expand_relpaths=False)
 
-        if self.cfg['m32']:
-            nlspath = os.path.join('idb', '32', 'locale', '%l_%t', '%N')
-        else:
-            nlspath = os.path.join('idb', 'intel64', 'locale', '%l_%t', '%N')
-        txt += self.module_generator.prepend_paths('NLSPATH', nlspath)
-
```

Comment:
The debugger is nowadays a version of gdb and the locale is under
debugger_2018/gdb/intel64/share/locale/

But I think we can scrap the NLSPATH completely as above for debugger.

@boegel @damianam Thoughts?

Reply:
I agree